js/script.js

```diff
@@ -1,8 +1,16 @@
-$(function () {
-    $(document).scroll(function () {
-        var $nav = $(".fixed-top");
-        $nav.toggleClass('scrolled', $(this).scrollTop() > $nav.height());
-        var $text = $(".text-white");
+$(function() {
+    $(document).scroll(function() {
+        var $nav = $('.fixed-top')
+        $nav.toggleClass('scrolled', $(this).scrollTop() > $nav.height())
+        var $text = $('.text-white')
         $text.toggleClass('dark', $(this).scrollTop() > $nav.height())
-    });
-});
\ No newline at end of file
+    })
+    $('.navbar-toggler').click(() => {
+        var $nav = $('.fixed-top')
+        var $text = $('.text-white')
```

Comment:
The `.text-white` class will be used by many other elements. Will it be better to use an id to refer to the elements?

Reply:
yes